package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "express-enforces-ssl",
-  "version": "1.1.0",
+  "name": "express-enforces-ssl-update",
```

Comment:
Can I kindly ask why would you want to change the package name? It's enough to do only versioning update and code tweaks. If any project has a ci/cd package publisher and automation, it will fail btw. My suggestion is to leave it as it is.

Reply:
Because this project hasn't seen an update in 9 years, I chose to just publish my updates to npm and simply move on... I didn't even see your replies until now. 

For anyone stumbling across this:
`npm install express-enforces-ssl-update`